### ISSUE TITLE
Phone rotation exits chat room

### ISSUE THREAD

**Issue by voodooboy3000** [numixproject/org.numixproject.hermes#32]
I just noticed when I rotate my phone while in a room I get kicked back to the server details screen.

**Comment by nanepiwo** [comment]
Yes, the client should support both landscape and portrait in all modes but it doesn't.
